### PR TITLE
Create signal-desktop.profile

### PR DESCRIPTION
**Pull request by onny**
Profile for signal-desktop (standalone electron app, see https://github.com/WhisperSystems/Signal-Desktop)

### CODE REVIEW

File: etc/signal-desktop.profile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Persistent local customizations
+include /etc/firejail/signal-desktop.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+noblacklist ~/.config/Signal
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-programs.inc
```
Comment (SpotComms) [inline etc/signal-desktop.profile]: What about `include /etc/firejail/disable-passwdmgr.inc`?

File: etc/signal-desktop.profile [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Firejail profile for sinal-desktop
```
Comment (SpotComms) [inline etc/signal-desktop.profile]: "sinal" -> "signal"

File: etc/signal-desktop.profile [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-programs.inc
+
+mkdir ~/.config/Signal
+whitelist ${DOWNLOADS}
+whitelist ~/.config/Signal
+include /etc/firejail/whitelist-common.inc
+
```
Comment (SpotComms) [inline etc/signal-desktop.profile]: If you're on 0.9.51 can you test if `include /etc/firejail/whitelist-var-common.inc` works?

File: etc/signal-desktop.profile (hunk 1 above)
Comment (onny) [reply etc/signal-desktop.profile]: Yes, we can include this

File: etc/signal-desktop.profile (hunk 3 above)
Comment (onny) [reply etc/signal-desktop.profile]: Seems to work fine

File: etc/signal-desktop.profile [hunk 4]
```diff
@@ -32,4 +34,3 @@ private-dev
 private-tmp
 
 noexec ${HOME}
-noexec /tmp
```
Comment (Fred-Barclay) [inline etc/signal-desktop.profile]: Should we keep `noexec /tmp`?

Comment (onny) [reply etc/signal-desktop.profile]: Signal doesn't seem to work with noexec /tmp, don't know why

**Comment by SpotComms** [comment]
Merged, thanks!

**Pull request merged**